src/main/kotlin/com/sourcegraph/cody/chat/ContextFilesMessage.kt

```diff
@@ -35,7 +36,7 @@ class ContextFilesMessage(val project: Project, contextMessages: List<ContextMes
               if (it.repoName != null) {
                 "${project.basePath}/${it.uri.path}"
               } else {
-                it.uri.path
+                Paths.get(it.uri).absolutePathString()
```

Comment:
so that macos & windows paths are resolved correctly (thanks @pkukielka for this idea)